crates/transaction-generator-lib/src/publishing/module_simple.rs

```diff
@@ -292,6 +304,32 @@ impl EntryPoints {
                     bcs::to_bytes(count).unwrap(),
                 ])
             },
+            EntryPoints::MakeOrChangeTable { offset, count } => get_payload(
+                module_id,
+                ident_str!("make_or_change_table").to_owned(),
+                vec![
+                    bcs::to_bytes(offset).unwrap(),
+                    bcs::to_bytes(count).unwrap(),
+                ],
+            ),
+            EntryPoints::MakeOrChangeTableRandom {
```

Comment:
Silly question: instead of duplicating `MakeOrChangeTable`, would having a `bool` inside of the `MakeOrChangeTable` enum, e.g., `random`, enough to determine which mode to use?

Reply:
Should I combine MakeOrChangeTable and MakeOrChangeTableRandom into a single transaction as per your comment? The semantics of offset and count is different in both the transactions though.